lnwallet/channel.go

```diff
@@ -1328,12 +1345,44 @@ func (lc *LightningChannel) fetchCommitmentView(remoteChain bool,
 		numHTLCs++
 	}
 
+	// Initiate feePerKw to the last committed fee for this chain.
+	feePerKw := commitChain.tail().feePerKw
+
+	// Check if any fee updates have taken place since that last commitment.
+	if lc.channelState.IsInitiator {
```

Comment:
Since the "queue logic" will be the same as for any other `update_` message, I think we should treat the fee updates as any other htlc when we revisit this.